library/thrift

```diff
@@ -6,4 +6,6 @@
 0.10.0: git://github.com/ahawkins/docker-thrift@e1f81dfe3e8fac5588e12d2b880166d1743dbecd 0.10
 0.11: git://github.com/ahawkins/docker-thrift@00e197d889426695446baec4e034b5ddbb15bcb3 0.11
 0.11.0: git://github.com/ahawkins/docker-thrift@00e197d889426695446baec4e034b5ddbb15bcb3 0.11
-latest: git://github.com/ahawkins/docker-thrift@00e197d889426695446baec4e034b5ddbb15bcb3 0.11
+0.12: git://github.com/ahawkins-docker-thrift@d7e73876549d205898b1922bcbf69e74ef11c288 0.12
+0.12.0: git://github.com/ahawkins-docker-thrift@d7e73876549d205898b1922bcbf69e74ef11c288 0.12
+latest: git://github.com/ahawkins-docker-thrift@d7e73876549d205898b1922bcbf69e74ef11c288 0.12
```

Comment:
It would appear there's a typo here -- this probably should've been `ahawkins/docker-thrift` instead of `ahawkins-docker-thrift`. :sweat_smile: